LibSource/ComplexFloatArray.h

```diff
@@ -2,11 +2,17 @@
 #define __ComplexFloatArray_h__
 
 #include "FloatArray.h"
+#include "AudioBuffer.h"
```

Comment:
I'm not sure I like the added depencendy on AudioBuffer.
How much of a convenience is it to use `cplx.copyTo(buffer)` instead of 
```
getRealValues(buffer.getSamples(0));
getComplexValues(buffer.getSamples(1))
```

This has the advantage of being very explicit.

Reply:
yes I like that: `copyTo(FloatArray, FloatArray)`
If you can change that and the add/subtract issues then I'll merge it.